build.gradle

```diff
@@ -2,6 +2,43 @@
  * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
  */
 
+plugins {
+  id "com.jfrog.artifactory"
+  id "idea"
+}
+
+group = 'com.linkedin.cruisecontrol'
+
+project.ext {
+  pomConfig = {
+    url "https://github.com/linkedin/cruise-control"
+    licenses {
+      license {
+        name "BSD 2-CLAUSE LICENSE"
+        url "https://opensource.org/licenses/BSD-2-Clause"
+      }
+    }
+    developers {
+      developer {
+        name "Adem Efe Gencer"
+        email "efegencer@gmail.com"
+      }
+      developer {
+        name "Jiangjie (Becket) Qin"
+        email "becket.qin@gmail.com"
+      }
+      developer {
+        name "Sir Joel Koshy"
+        email "jjkoshy@yahoo.com"
+      }
+    }
+    scm {
+      url "https://github.com/linkedin/cruise-control"
+    }
+  }
+}
+
+
 allprojects {
   apply plugin: 'idea'
   apply plugin: 'eclipse'
```

Comment:
does anyone really need the eclipse plugin ?!

Reply:
Removed the plugin.